<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TDialogAddItem</class>
 <widget class="QDialog" name="TDialogAddItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование элемента</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../share/mainResources.qrc">
    <normaloff>:/icons/images/test.png</normaloff>:/icons/images/test.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Параметры</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Название</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="IdentificatorEdit" name="leName"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Краткое описание</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="shortDescr"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Описание</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="TextEdit" name="textEdit" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="toolTip">
        <string>Сохранить и выйти</string>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
       <property name="icon">
        <iconset resource="../../share/mainResources.qrc">
         <normaloff>:/textedit/images/filesave.png</normaloff>:/textedit/images/filesave.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="toolTip">
        <string>Выход без сохранения</string>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
       <property name="icon">
        <iconset resource="../../share/mainResources.qrc">
         <normaloff>:/icons/images/new/gtk-no.png</normaloff>:/icons/images/new/gtk-no.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEdit</class>
   <extends>QWidget</extends>
   <header location="global">textedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IdentificatorEdit</class>
   <extends>QLineEdit</extends>
   <header>identificatoredit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../share/mainResources.qrc"/>
 </resources>
 <connections/>
</ui>
